example_test.go

```diff
@@ -48,6 +49,16 @@ func ExampleNewTracer() {
 	// Output:
 }
 
+func ExampleNewTracer_tests() {
```

Comment:
what is the purpose of this function?

Reply:
The idea behind this was to have a new section in `godoc` for *New Tracer (tests)* showing how can you start a tracer to use in tests.

Reply:
Add the new section for godoc or remove this and take it to a next PR. Otherwise it dangles around without context.